app/src/main/java/com/duckduckgo/app/settings/db/SettingsDataStore.kt

```diff
@@ -146,6 +159,10 @@ class SettingsSharedPreferences @Inject constructor(private val context: Context
         return currentlySelected == clearWhenOption
     }
 
+    override fun isCurrentlySelected(fireAnimation: FireAnimation): Boolean {
```

Comment:
Consider renaming to `isFireAnimationCurrentlySelected` or something similar to give a bit more context since is kinda confusing  with the other `isCurrentlySelected` methods.

Reply:
Not sure about this one.

So, you have your settings, and you want to know if something is selected using `isCurrentlySelected`. Even tho there are currently 3 methods, the argument type will tell you what you ar checking. 

`isCurrentlySelected(fireAnimation)`is telling you that you are checking if that fire animation is selected inside settings.

adding `isFireAnimationCurrentlySelected(fireAnimation)` has the same result to me but longer. WDYT?

Reply:
Is like `selectedFireAnimationSavedValue` or `automaticallyClearWhatSavedValue` where we basically follow the opposite approach, the return types there tell us what it is but we are still adding a more meaningful function name.

Reply:
That's slightly different. You will get a compile error declaring two methods with the same name and the same arguments even if they return different types.

But wanted to make clear that this is just a personal preference, and I'm pretty sure I'm not even consistent with myself 😅, so I'm ok changing the name too. I think my main issue was the name being too long in that case.